### PR TITLE
feat: implement fragments

### PR DESCRIPTION
**Pull request by aizensoosuke**
Implements #4 - a very simple version use of fragments. The result is the UI below.

A hamburger in the action bar toggles the navigation drawer. 
Both items in the navigation drawer menu trigger a fragment swap in the main activity when you click on them. 
The fragments are designed as xml files directly, and not using jetpack compose.

### CODE REVIEW

**Comment by aizensoosuke** [comment]
Please do not merge this. Just review it and then we'll decide as a team if we want to merge :) Thank you !

**Review by lipefree — CHANGES_REQUESTED** [review]
Just need testings, otherwise LGTM

**Comment by aizensoosuke** [comment]
Won't do. We're using JetPack Compose, and it's not useful to work on this anymore.

**Pull request closed without merging**
